assets/js/src/core/components/grid/grid.tsx

```diff
@@ -52,6 +55,7 @@ export interface ColumnMetaType {
 declare module '@tanstack/react-table' {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   export interface ColumnMeta<TData extends RowData, TValue> extends ColumnMetaType {
+    size?: number
```

Comment:
Column meta does not have a size as the size is on top level of the column definition - should we remove this?

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/1477/commits/68e447c9fef6cc258e5211cc887663cde5e5ce51